src/mesh/mesh_generation.C

```diff
@@ -2752,5 +2752,44 @@ void MeshTools::Generation::build_delaunay_square(UnstructuredMesh & mesh,
 #endif // LIBMESH_HAVE_TRIANGLE && LIBMESH_DIM > 1
 
 
+void MeshTools::Generation::surface_octahedron
+  (UnstructuredMesh & mesh,
+   Real xmin, Real xmax,
+   Real ymin, Real ymax,
+   Real zmin, Real zmax,
+   bool flip_tris)
```

Comment:
`MeshTools::Generation::surface_octahedron()` seems a little bit specific? I could see having a `MeshTools` implementation for triangulating a regular polygon with n sides... but this looks more like unit test code to me.

Reply:
I'm actually good with having the ability to mesh 3D *octahedrons* in MeshTools::Generation. For some reason, the words "surface" and "octahedron" combined in my brain to equal "octagon".